app/controllers/players_controller.rb

```diff
@@ -9,6 +9,7 @@ def new
   def create
     @player = Player.new(player_params)
     @player.game_id = @game.id
+    if no_arthur? then @player.arthur = true end
```

Comment:
a shorter one-line syntax for conditionals: `@player.arthur = true if no_arthur?`

Reply:
So whoever joins the game first - i.e. whoever creates the game - has to be Arthur?

Perhaps it would be better to assign Arthurness when the game is quorate.

Reply:
That would also allow us to write the assignment as a callback on the model (to run whenever a player is associated with a game), again keeping the controller 'thin' - not that I've been especially careful about a thin controller myself.

Reply:
Yeah, that sounds plausible (particularly moving it to the model). I thought that having it given to the first player means that there is always a player with Arthur, but at the moment there is no way to assign it. The first player could also leave and a new player would have to join to get a new Arthur.

Reply:
At the moment, I am having the game-creator specify the number of players in advance, with 3 minimum. I wonder if I should validate 3 minimum in the database, in the form model, or in the form field? Form model feels like a happy medium. Where would you do such a validation if doing a Python app?